TwitchLib.Api.Core/Undocumented/Undocumented.cs

```diff
@@ -170,16 +170,16 @@ public Task<ChatUserResponse> GetChatUserAsync(string userId, string channelId =
 
         #region IsUsernameAvailable
 
-        public Task<bool> IsUsernameAvailableAsync(string username)
+        public async Task<bool> IsUsernameAvailableAsync(string username)
         {
             var getParams = new List<KeyValuePair<string, string>> {new KeyValuePair<string, string>("users_service", "true")};
-            var resp = RequestReturnResponseCode($"https://passport.twitch.tv/usernames/{username}", "HEAD", getParams);
+            var resp = await RequestReturnResponseCode($"https://passport.twitch.tv/usernames/{username}", "HEAD", getParams);
```

Comment:
just one more ConfigureAwait(false) here

Reply:
x.x

I rly need to setup analyzers for that xD